### PR TITLE
[Doc] Split data model and update related proper names

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Review by amber-create — DISMISSED** [review]

File: docs/TOC.md [hunk 1]
```diff
@@ -15,7 +15,12 @@
   + [Load and query data](./quick_start/Import_and_query.md)
 + Table Design
   + [Understand StarRocks table design](./table_design/StarRocks_table_design.md)
-  + [Data models](./table_design/Data_model.md)
+  + Table types
+    + [Table types](./table_design/table_types/table_types.md)
+    + [Duplicate key table](./table_design/table_types/duplicate_key_table.md)
```
Comment (amber-create) [inline docs/TOC.md:20]: 这个 key 大写 Key 吧，下面几个模型同理。

File: docs/administration/Profiling.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
 ## Table Creation
 
-### Data Model Selection
+### Table Type Selection
 
-StarRocks supports three kinds of data model: AGGREGATE KEY, UNIQUE KEY, and DUPLICATE KEY. All three are sorted by KEY.
+StarRocks supports four table types: Duplicate Key table, Aggragate Key table, Unique Key table, and Primary Key table. All three are sorted by KEY.
```
Comment (amber-create) [inline docs/administration/Profiling.md:7]: image
聚合模型的英文我记得是说不加 Key 了，写 Aggregate table

File: docs/administration/Profiling.md [hunk 3]
```diff
@@ -2,11 +2,11 @@
...
 
-* AGGREGATE KEY: When the AGGREGATE KEY is the same, the old and new records are aggregated. The currently supported aggregate functions are SUM, MIN, MAX, REPLACE. Aggregate  model can aggregate data in advance, which is suitable for reporting and multi-dimensional analyses.
+* AGGREGATE KEY: When the AGGREGATE KEY is the same, the old and new records are aggregated. The currently supported aggregate functions are SUM, MIN, MAX, REPLACE. Aggregate table can aggregate data in advance, which is suitable for reporting and multi-dimensional analyses.
```
Comment (amber-create) [inline docs/administration/Profiling.md:9]: image
加个 连词 and

Comment (amber-create) [inline docs/administration/Profiling.md:9]: image
1）冠词/单复数。这里是非特指，可以用 An Aggregate table 或者 Aggregate tables。
2）Aggregate table 作为行动主体执行 aggregate 动作吗？感觉不太合理，应该是 Aggregate table 支持 （support）aggregating data in advance （或者 写 supports upfront data aggregations）。

Comment (amber-create) [inline docs/administration/Profiling.md:9]: image
建议改为 -> facilitating 类似的

File: docs/administration/information_schema.md [hunk 4]
```diff
@@ -82,7 +82,7 @@ The following fields are provided in `tables_config`:
 | TABLE_SCHEMA     | Name of the database that stores the table.                  |
 | TABLE_NAME       | Name of the table.                                           |
 | TABLE_ENGINE     | Engine type of the table.                                    |
-| TABLE_MODEL      | Data model of the table. Valid values: "DUP_KEYS", "AGG_KEYS", "UNQ_KEYS" or "PRI_KEYS". |
+| TABLE_MODEL      | Table type. Valid values: "DUP_KEYS", "AGG_KEYS", "UNQ_KEYS" or "PRI_KEYS". |
```
Comment (amber-create) [inline docs/administration/information_schema.md:85]: image
The table type of the table?

File: docs/faq/Others.md [hunk 5]
```diff
@@ -57,7 +57,7 @@ ALTER DATABASE example_db SET DATA QUOTA 10T;
 
 ## Does StarRocks support updating particular fields in a table by executing the UPSERT statement?
 
-StarRocks 2.2 and later support updating specific fields in a table by using the primary key model. StarRocks 1.9 and later support updating all fields in a table by using the primary key model. For more information, see [Primary key model](../table_design/Data_model.md#primary-key-model) in StarRocks 2.2.
+StarRocks 2.2 and later support updating specific fields in a table by using the Primary Key table. StarRocks 1.9 and later support updating all fields in a table by using the Primary Key table. For more information, see [Primary key table](../table_design/table_types/primary_key_table.md) in StarRocks 2.2.
```
Comment (amber-create) [inline docs/faq/Others.md:60]: image
大写为 Key

File: docs/table_design/Materialized_view.md [hunk 6]
```diff
@@ -22,7 +22,7 @@ The two mentioned use cases are both valid since the users need to:
...
+In the existing StarRocks table type, if you create only one table with aggregation model, for example a table with `item_id`, `sold_time`, `customer_id`, `sum(price)`, you are not able to analyze detailed data since the aggregation drops some information of the data. If only a duplicate model is built, you can run queries on all the dimension, but the queries will not be accelerated because of lack of Rollup support. If you build a table with the aggregation model and a table with the duplicate model at the same time, you can get both the performance and can run queries on all the dimensions, but the two tables are not related to each other, so you need to choose the analysis table manually. It doesn’t offer good flexibility and usability.
```
Comment (amber-create) [inline docs/table_design/Materialized_view.md:25]: image
这里还有 model 没改到

Comment (amber-create) [reply docs/table_design/Materialized_view.md:25]: image
model

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.0 branch-2.5 branch-2.4

**Pull request merged**